html/sqlvalidator/sqlvalidator.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>SQL Validator Index page</title>
```

Comment:
Would prefer this to be more generic, i.e. `validator` and file path like `validator/validator.html` as I want to add swagger and others to this